client/src/components/CKANResultConditionsField.js

```diff
@@ -155,7 +155,28 @@ class CKANResultConditionsField extends Component {
     );
   }
 
+  /**
+   * Render a simple <p> tag for read only mode
+   *
+   * @returns {HTMLElement|null}
+   */
+  renderReadOnly() {
+    const { data: { source } } = this.props;
+    const input = this.getInputValue();
+    const type = source.find(candidate => candidate.value === this.getSelectValue()).title;
+
+    if (!type || !input) {
+      return null;
```

Comment:
This is a little weird in the UI in my opinion. Would be better to render an empty UI control maybe

Reply:
Question: any reason we are using a p tag instead of readonly text inputs?

Reply:
I considered that but it looked weirder than just not having it 🤷‍♂️ . I went over the designs with Sacha but _not_ what happens when you're looking at a read only version with nothing set.

Reply:
Ok I've altered it now. The field on the PHP side will delegate to the existing handler when the value is empty.